pkg/client/events_query/client_test.go

```diff
@@ -4,84 +4,71 @@ import (
 	"context"
```

Comment:
:disappointed: Mistakes were make; this file was already committed in ad0121d63e1a219e551765229bf907731ab28929.